nixos/modules/misc/ids.nix

```diff
@@ -288,6 +288,7 @@
       kresd = 270;
       rpc = 271;
       geoip = 272;
+      slimserver = 273;
```

Comment:
I didn't know about dynamic ids yet. Maybe it would make sense to put a comment in the ids file that dynamic ids are available? Best placed at the end of the list where one would add a new entry?